CHANGELOG.md

```diff
@@ -1,3 +1,39 @@
+## Release 0.7.0
+
+New features:
+- Track short-lived connections
+  [#356](https://github.com/weaveworks/scope/issues/356)
+- Details pane: add a lot more information and present it in a more intuitive way.
+  [#394](https://github.com/weaveworks/scope/pull/394)
+  [#396](https://github.com/weaveworks/scope/pull/396)
+  [#400](https://github.com/weaveworks/scope/pull/400)
+  [#387](https://github.com/weaveworks/scope/pull/387)
+  [#370](https://github.com/weaveworks/scope/issues/370)
+  [#322](https://github.com/weaveworks/scope/issues/322)
+- Make edge direction flow from client to server.
+  [#355](https://github.com/weaveworks/scope/pull/355)
+- Make pseudo nodes clickable.
```

Comment:
Show more information for pseudo nodes where possible - such as processes for uncontained, and connects to/from the internet.